doc/source/whatsnew/v0.18.2.txt

```diff
@@ -152,3 +152,4 @@ Bug Fixes
 - Bug in ``Period`` addition raises ``TypeError`` if ``Period`` is on right hand side (:issue:`13069`)
 - Bug in ``Peirod`` and ``Series`` or ``Index`` comparison raises ``TypeError`` (:issue:`13200`)
 - Bug in ``pd.set_eng_float_format()`` that would prevent NaN's from formatting (:issue:`11981`)
+- Bug in ``Float64Index.astype(int)`` raises ``ValueError`` if ``NaN`` is converted to integer; furthermore ``TimedeltaIndex``, ``DatetimeIndex`` return ``Index64Int`` on ``.astype(int)`` (:issue:`13149`)
```

Comment:
so you can list the second point as a separate bug fix issue and use the PR number as the issue number